acceptance_tests/features/pages/collection_exercise_details.py

```diff
@@ -30,6 +30,18 @@ def get_collection_instrument_success_text():
     return browser.find_by_id('collection-instrument-success').text
 
 
+def click_survey_breadcrumb():
+    browser.find_by_id('breadcrumb-2').click()
+
+
+def get_breadcrumbs():
```

Comment:
Maybe the breadcrumb stuff could live in its own package for breadcrumbs if it's used on all or most pages